examples/parameterized-routing/server.js

```diff
@@ -0,0 +1,27 @@
+const { createServer } = require('http')
+const { parse } = require('url')
+const next = require('next')
+const pathMatch = require('path-match')
+
+const app = next({ dev: true })
```

Comment:
Based on https://github.com/zeit/next.js/pull/310/files#r92954990, looking at `./dist/server/index.js` the api seems to require `new next({...})` to create the server.